controllers/controller.js

```diff
@@ -79,3 +82,11 @@ exports.sendComment = (req, res, next) => {
 		})
 		.catch(next);
 };
+
+exports.removeComment = (req, res, next) => {
+	const commentId = req.params.comment_id;
+
+	deleteComment(commentId).then(() => {
+		res.status(204).send();
```

Comment:
There is a cheeky `res.sendStatus` method for explicitly not having a response body